src/Utilities/StdHelpers.hpp

```diff
@@ -389,3 +389,18 @@ inline std::array<T, Dim> operator-(const std::array<T, Dim>& rhs) noexcept(
   }
   return result;
 }
+
```

Comment:
This will conflict with the addition of magnitude in the Wedge2D PR. We should probably decide which version we want. I personally think I prefer the way this is written to the way mine is, but that we should also allow it to work for DataVectors too.